app/globals.css

```diff
@@ -74,3 +74,7 @@
     @apply bg-background text-foreground;
   }
 } */
+
+.debug {
```

Comment:
Remove this. We don't want users to be able to add this class in the browser.

Reply:
Sorry, that was my doing for testing purposes.